examples/test-unknowns/ts/index.ts

```diff
@@ -1,23 +1,22 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as docker from "@pulumi/docker";
-import * as random from "@pulumi/random";
+import * as command from "@pulumi/command";
 
 
 
-const randName = new random.RandomString("random", {
-    length: 10,
-
+const randArg = new command.local.Command ("arg", {
+   create: "echo setMyArg"
 });
 
 const img = new docker.Image("docker-565-one", {
     imageName: "pulumibot/test-image:with-build-args",
 
     build: {
         args: {
-            "RANDOM_ARG": randName.id
+            "RANDOM_ARG": randArg.stdout
```

Comment:
Do you want a preview test against this stack as well since you've fixed the inputs to actually be unknown now?

Reply:
oh that's a great idea!

Reply:
added in subsequent commit!